OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1876,6 +1927,19 @@ public static DreamValue NativeProc_splittext(DreamObject instance, DreamObject
             return new DreamValue(format);
         }
 
+        [DreamProc("trunc")]
+        [DreamProcParameter("n", Type = DreamValueType.Float)]
+        public static DreamValue NativeProc_trunc(DreamObject instance, DreamObject usr, DreamProcArguments arguments) {
+            DreamValue arg = arguments.GetArgument(0, "n");
+            if (arg.TryGetValueAsFloat(out float floatnum)) {
+                if (float.IsInfinity(floatnum)) {
+                    return new DreamValue(0);
+                }
```

Comment:
`trunc(1.#INF)` returns `1.#INF`

Reply:
whoops lol, though it would be the same as fract but nope